benchmark/browser/scenarios/sx-prop-box-material-ui/index.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import { logReactMetrics } from '../utils';
+
+const theme = createMuiTheme();
+
+export default function BasicStyledComponents() {
+    return (
+      <React.Profiler id="sx-prop-box-material-ui" onRender={logReactMetrics}>
```

Comment:
The more tests we add, the more I think we could move the profiler into the benchmark wrapper (have it once, not duplicated in each test case).

Reply:
Done in https://github.com/mui-org/material-ui/pull/23178 together with the output in README.md